Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -64,14 +66,63 @@ public Context getContext() {
     @SuppressLint("SetTextI18n")
     public void onViewBound(View view) {
         super.onViewBound(view);
-
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
+        int padding = Utils.getDefaultPadding();
+        int p = padding / 2;
+        File folder = new File(Constants.BASE_PATH, "crashlogs");
+        File[] files = folder.listFiles();
+        assert files != null;
+
+        AppCompatImageButton crashFolderBtn = new AppCompatImageButton(context);
+        int ic1 = R$d.ic_open_in_new_white_24dp;
+        AppCompatImageButton clearLogsBtn = new AppCompatImageButton(context);
+        int ic2 = R$d.ic_delete_white_24dp;
+
+        Toolbar.LayoutParams crashFolderBtnParams = new Toolbar.LayoutParams(Toolbar.LayoutParams.WRAP_CONTENT, Toolbar.LayoutParams.WRAP_CONTENT);
+        crashFolderBtnParams.gravity = Gravity.END;
+        crashFolderBtnParams.setMarginEnd(p);
+        crashFolderBtn.setLayoutParams(crashFolderBtnParams);
+        Toolbar.LayoutParams clearLogsParams = new Toolbar.LayoutParams(Toolbar.LayoutParams.WRAP_CONTENT, Toolbar.LayoutParams.WRAP_CONTENT);
+        clearLogsParams.gravity = Gravity.END;
+        clearLogsBtn.setLayoutParams(clearLogsParams);
+        crashFolderBtn.setPadding(p, p, p, p);
+        clearLogsBtn.setPadding(p, p, p, p);
+
+        crashFolderBtn.setBackgroundColor(Color.TRANSPARENT);
+        clearLogsBtn.setBackgroundColor(Color.TRANSPARENT);
+
+        //noinspection ConstantConditions
+        Drawable openCrashesExternal = ContextCompat.getDrawable(context, ic1).mutate();
+        if (folder.exists()) openCrashesExternal.setAlpha(185);
+        if (!folder.exists()) openCrashesExternal.setAlpha(92);
+        crashFolderBtn.setImageDrawable(openCrashesExternal);
+        //noinspection ConstantConditions
+        Drawable clearLogs = ContextCompat.getDrawable(context, ic2).mutate();
+        clearLogs.setAlpha(185);
+        if (files.length == 0) clearLogs.setAlpha(92);
+        clearLogsBtn.setImageDrawable(clearLogs);
+
+        crashFolderBtn.setOnClickListener(e -> {
+            if (!folder.exists()) return;
```

Comment:
it would be better UX to create the folder if it doesnt exist and always show this button